src/Jss.js

```diff
@@ -45,9 +45,19 @@ export default class Jss {
    * Create a Style Sheet.
    */
   createStyleSheet(styles: Object, options: StyleSheetOptions): StyleSheet {
+    let generateClassName
+    if (options
+      && options.sheetsRegistryContext
+      && this.options.generateClassName === generateClassNameDefault) {
```

Comment:
not sure if this logic will work, for e.g. when user defined already a different generateClassName